client/src/components/Reviews/ReviewsContainer.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import ReviewSorter from './ReviewSorter.jsx';
+import ReviewList from './ReviewList.jsx';
+
+const RVC = styled.div`
+margin-top: 10px;
+display: flex;
+flex-direction: row;
+justify-content: left;
+`;
+
+export default function ReviewsContainer (props) {
```

Comment:
Technically this reviewsContainer is essentially holding the RVC from above correct?

I'm down to keep it separated if it helps with code separation, but it would be possible to remove this level and create the RVC one component level up if you wanted.

Reply:
Good call--will implement that change in next PR